htdocs/modulebuilder/index.php

```diff
@@ -1570,11 +1570,13 @@
 
 llxHeader('', $langs->trans("ModuleBuilder"), '', '', 0, 0,
 	array(
-	'/includes/ace/src/ace.js',
-	'/includes/ace/src/ext-statusbar.js',
-	'/includes/ace/src/ext-language_tools.js',
-	//'/includes/ace/src/ext-chromevox.js'
-	), array(), '', 'classforhorizontalscrolloftabs');
```

Comment:
The classforhorizontalscrolloftabs is required to have the tabs/list having an horizontal scroll. It may be added everywhere in a future. 
According to the screenshot of the bug you mention, it seems you have another select box compared to standard version. Probably coming from an external module or experimental option. Fix should be done into this external module or option.